package.json

```diff
@@ -45,10 +45,11 @@
   },
   "devDependencies": {
     "@sinonjs/fake-timers": "^6.0.1",
-    "eslint": "2.13.1",
+    "eslint": "^8.10.0",
```

Comment:
why do we need to update eslint here ?
Seems like it should be a separate commit (or even PR)...